package.json

```diff
@@ -9,5 +9,11 @@
   },
   "devDependencies": {
     "lerna": "^5.5.0"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
```

Comment:
These dependencies shouldn't be here, we need to add them to the specific sub-project